### ISSUE TITLE
Html is broken in DateRangePicker

### ISSUE THREAD

**Issue by SDKiller** [2amigos/yii2-date-picker-widget#9]
Not sure when exactly it happened - maybe after bootstrap updated to 3.3.4

See screen - seems that class `form-control` for container is excessive (after removing it `input` takes its place, but also `input-group-addon` height need to be adjusted):

![qip shot - screen 612 31 03 15](https://cloud.githubusercontent.com/assets/2150916/6923221/06e5ceda-d7d9-11e4-91ef-379ee42910ed.png)

After removing:

![qip shot - screen 613 31 03 15](https://cloud.githubusercontent.com/assets/2150916/6923354/d34c56e2-d7d9-11e4-9a85-b0050947674f.png)

**Comment by tonydspaniard** [comment]
did you run composer update? what is the version  you use?

**Comment by SDKiller** [comment]
```
        "name": "2amigos/yii2-date-picker-widget",
        "version": "dev-master",
        "version_normalized": "9999999-dev",
        "source": {
            "type": "git",
            "url": "https://github.com/2amigos/yii2-date-picker-widget.git",
            "reference": "c61989074044185f35cb47379e8715b9729c5674"
        },
```

**Comment by tonydspaniard** [comment]
I have found the issue. Will fix it tonight asap.

**Comment by SDKiller** [comment]
Also had to remove from https://github.com/2amigos/yii2-date-picker-widget/blob/master/src/DateRangePicker.php#L91 and https://github.com/2amigos/yii2-date-picker-widget/blob/master/src/DateRangePicker.php#L99

**Comment by SDKiller** [comment]
I suppose  `input-group` should be in lines 91 and 99 instead, this fixed everything:

```php
        if ($this->form) {
            Html::addCssClass($this->options, 'datepicker-from');
            Html::addCssClass($this->optionsTo, 'datepicker-to');
            $inputFrom = $this->form->field(
                $this->model,
                $this->attribute,
                [
                    'template' => '{input}{error}',
                    'options' => ['class' => 'input-group datepicker-range'],
                ]
            )->textInput($this->options);
            $inputTo = $this->form->field(
                $this->model,
                $this->attributeTo,
                [
                    'template' => '{input}{error}',
                    'options' => ['class' => 'input-group datepicker-range'],
                ]
            )->textInput($this->optionsTo);
        } else {
```

**Comment by SDKiller** [comment]
Ah, sorry, forgot about `.form-control:first-child`. My fix results rounded corners for `$inputTo`

**Comment by tonydspaniard** [comment]
@SDKiller mind showing the fix?

**Comment by tonydspaniard** [comment]
i have pushed some changes... mind checking if that fixed it?

**Comment by SDKiller** [comment]
As I said earlier - it fixes position, but adding `input-group` affects corners of `$inputTo`:

![qip shot - screen 615 01 04 15](https://cloud.githubusercontent.com/assets/2150916/6934137/9e502536-d838-11e4-8dad-956dc291623f.png)

Had to adjust css also (nesessary only if `form` is specified in widget config):

```
.input-group > .datepicker-to.form-control:first-child {
    border-radius: 0 4px 4px 0;
}
```

Without `form` widget plays well without css adjustments.

**Comment by tonydspaniard** [comment]
The problem was on the assets not on the style. I have fixed, will update the version tag soon.

**Comment by tonydspaniard** [comment]
done... Fixed on 1.0.5 Thanks for the help!
